src/async_wrap.h

```diff
@@ -155,10 +160,12 @@ class AsyncWrap : public BaseObject {
   inline double get_trigger_async_id() const;
 
   void AsyncReset(v8::Local<v8::Object> resource,
-                  double execution_async_id = -1,
+                  bool skip_destroy,
+                  double execution_async_id = kInvalidAsyncId,
                   bool silent = false);
 
-  void AsyncReset(double execution_async_id = -1, bool silent = false);
```

Comment:
Actually this one should be private as it results in reuse of the resource but it is needed by `FileHandle::ReadStart()` which seems to reuse `FileHandleReadWrap` instances.